SimG4Core/MagneticField/interface/Field.h

```diff
@@ -3,14 +3,18 @@
 
 #include "G4MagneticField.hh"
 
-class MagneticField;
+#include "MagneticField/Engine/interface/MagneticField.h"
+
+#include "DataFormats/GeometryVector/interface/GlobalPoint.h"
+
+#include "CLHEP/Units/GlobalSystemOfUnits.h"
```

Comment:
@VinInn , this header should not be used in header file, we are trying do not use units in headers. Alternatively, use 

#include <CLHEP/Units/SystemOfUnits.h>

Reply:
I need to inline the method.